ResourceProvisioner/src/ResourceProvisioner_PyFunctions/function_app.py

```diff
@@ -130,8 +154,77 @@ def sync_storage_workspace_users_function(workspace_definition):
 
     # Cleanup users in workspace that aren't in AAD Graph
     #remove_deleted_users_in_workspace(workspace_client)
-    #synchronize_workspace_users(workspace_definition, workspace_client)    
+    #synchronize_workspace_users(workspace_definition, workspace_client) 
 
+    
+''' testing the keys_upper function
+test_dict = {
```

Comment:
this is exactly what I had in mind - why not move this to the unit test project?